### PR TITLE
✨ new connect page

### PR DESCRIPTION
**Pull request by okusindja**
- 💫 feat: add illustration and animation
- 🔥 chore: remove unused files

### CODE REVIEW

File: views/dapp/v2/components/layout/connect-wallet/wallet/wallet-list-section.tsx [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
...c4MzhDNTQuNTg0OCA0Ny4xOTI2IDU0LjU4NDggNDAuNTE2MyA1MS4zNTY0IDM0LjkyNEwzNy43NTI0IDExLjM2MTVDMzYuOTI0MSA5LjkyNzAxIDM0Ljg1MzUgOS45MjcwMSAzNC4wMjUzIDExLjM2MTVMMjAuNDIxMyAzNC45MjRDMTcuMTkyOSA0MC41MTUyIDE3LjE5MjkgNDcuMTkxNSAyMC40MjEzIDUyLjc4MzhaTTMyLjA1NjYgMjIuNTcxM0wzNC45NTcxIDE3LjU0NzRDMzUuMzcxMiAxNi44MzAxIDM2LjQwNjUgMTYuODMwMSAzNi44MjA2IDE3LjU0NzRMNDcuOTc5MSAzNi44NzQ4QzUwLjAyOTEgNDAuNDI1NCA1MC40MTM5IDQ0LjUzNSA0OS4xMzM1IDQ4LjI5NTRDNDkuMDAwMiA0Ny42ODE5IDQ4LjgxMzggNDcuMDU0MiA0OC41NjI2IDQ2LjQyMDFDNDcuMDIxMyA0Mi41MzA0IDQzLjUzNjMgMzkuNTI4OSAzOC4yMDIzIDM3LjQ5ODJDMzQuNTM1MSAzNi4xMDcxIDMyLjE5NDMgMzQuMDYxMyAzMS4yNDMxIDMxLjQxNzFDMzAuMDE4IDI4LjAwODkgMzEuMjk3NiAyNC4yOTI0IDMyLjA1NjYgMjIuNTcxM1pNMjcuMTEwNyAzMS4xMzc5TDIzLjc5ODYgMzYuODc0OEMyMS4yNzQ4IDQxLjI0NTkgMjEuMjc0OCA0Ni40NjQxIDIzLjc5ODYgNTAuODM1M0MyNi4zMjIzIDU1LjIwNjQgMzAuODQxMyA1Ny44MTUgMzUuODg4OCA1Ny44MTVDMzkuMjQxMyA1Ny44MTUgNDIuMzYxNSA1Ni42NjMzIDQ0LjgxODQgNTQuNjA4OEM0NS4xMzg4IDUzLjgwMjEgNDYuMTMxIDUwLjg0OTIgNDQuOTA1MiA0Ny44MDU4QzQzLjc3MyA0NC45OTU0IDQxLjA0ODIgNDIuNzUxOSAzNi44MDYxIDQxLjEzNkMzMi4wMTEgMzkuMzE3MSAyOC44OTU4IDM2LjQ3NzQgMjcuNTQ4NiAzMi42OTg0QzI3LjM2MzEgMzIuMTc4MSAyNy4yMTg5IDMxLjY1NjggMjcuMTEwNyAzMS4xMzc5WiIgZmlsbD0id2hpdGUiLz4KPC9zdmc+';
```
Comment (okusindja) [inline views/dapp/v2/components/layout/connect-wallet/wallet/wallet-list-section.tsx:19]: add to constants or as a separated file

**Review by okusindja — COMMENTED** [review]
approved @KipandaJr

File: assets/messages/connect-wallet/en-US.json
Comment (git-marcopitra) [inline assets/messages/connect-wallet/en-US.json:1]: Move it to common

File: assets/messages/connect-wallet/es-ES.json
Comment (git-marcopitra) [inline assets/messages/connect-wallet/es-ES.json:1]: Move it to common

File: assets/messages/connect-wallet/pt-BR.json
Comment (git-marcopitra) [inline assets/messages/connect-wallet/pt-BR.json:1]: Move it to common

File: assets/messages/connect-wallet/pt-PT.json
Comment (git-marcopitra) [inline assets/messages/connect-wallet/pt-PT.json:1]: Move it to common

File: interface/declaration.d.ts
Comment (git-marcopitra) [inline interface/declaration.d.ts:1]: We don't need these changes

File: pages/dapp/alpha/faucet/index.tsx
Comment (git-marcopitra) [inline pages/dapp/alpha/faucet/index.tsx:1]: Revert these changes

File: pages/dapp/alpha/lending/index.tsx
Comment (git-marcopitra) [inline pages/dapp/alpha/lending/index.tsx:1]: Revert these changes

File: pages/dapp/dex/pool/details.tsx
Comment (git-marcopitra) [inline pages/dapp/dex/pool/details.tsx:1]: Revert these changes

File: pages/dapp/dex/pool/find.tsx
Comment (git-marcopitra) [inline pages/dapp/dex/pool/find.tsx:1]: Revert these changes

File: pages/dapp/dex/pool/index.tsx
Comment (git-marcopitra) [inline pages/dapp/dex/pool/index.tsx:1]: Revert these changes

File: pages/dapp/create-token/index.tsx
Comment (git-marcopitra) [inline pages/dapp/create-token/index.tsx:1]: Revert the connect wallet changes

File: pages/dapp/farms/details.tsx
Comment (git-marcopitra) [inline pages/dapp/farms/details.tsx:1]: Revert the changes

File: pages/dapp/farms/index.tsx
Comment (git-marcopitra) [inline pages/dapp/farms/index.tsx:1]: Revert the changes

File: pages/dapp/liquidity/details.tsx
Comment (git-marcopitra) [inline pages/dapp/liquidity/details.tsx:1]: Revert the changes

File: pages/dapp/liquidity/index.tsx
Comment (git-marcopitra) [inline pages/dapp/liquidity/index.tsx:1]: Revert the changes

File: pages/dapp/swap/index.tsx
Comment (git-marcopitra) [inline pages/dapp/swap/index.tsx:1]: Revert the changes

File: views/dapp/v2/components/layout/connect-wallet/illustration/illustration-section.tsx [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+import { not } from 'ramda';
+import { FC } from 'react';
+
+import { TimesSVG } from '@/components/svg/v2';
+
+import { IllustrationProps } from '../connect-wallet.types';
+import ConnectIllustration from './illustration';
+
+const IllustrationSection: FC<IllustrationProps> = ({ setOpenWallet }) => {
+  const menuVariants = {
```
Comment (git-marcopitra) [inline views/dapp/v2/components/layout/connect-wallet/illustration/illustration-section.tsx:11]: Keep this constant outside the component, to not be rendered all the time again

File: views/dapp/v2/components/layout/connect-wallet/illustration/illustration-section.tsx [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+  };
+
+  return (
+    <Box
+      display={['none', 'none', 'none', 'flex']}
+      color="text"
+      overflowY="auto"
+      maxHeight="100vh"
+      height="100vh"
+      width={['100%', '100%', '100%', '50%']}
```
Comment (git-marcopitra) [inline views/dapp/v2/components/layout/connect-wallet/illustration/illustration-section.tsx:29]: why 100% if u are not using others displays? just put it all as 50%

File: views/dapp/v2/components/layout/connect-wallet/illustration/illustration.tsx [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+
+import {
+  BodyConnectorSVG,
+  ConnectConnectorFragSVG,
+  ConnectStructureSVG,
+} from '@/components/svg/v2';
+import HeroBlock from '@/views/institutional/components/hero/hero-block';
+
+const ConnectIllustration: FC = () => {
+  return (
```
Comment (git-marcopitra) [inline views/dapp/v2/components/layout/connect-wallet/illustration/illustration.tsx:12]: remove the return

File: views/dapp/v2/components/layout/connect-wallet/wallet/wallet-item.tsx [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+import { WalletItemProps } from '../connect-wallet.types';
+import WalletItemButtons from './wallet-item-buttons';
+
+const WalletItem: FC<WalletItemProps> = ({
+  icon,
+  name,
+  hasInstalled,
+  openWalletModal,
+}) => {
+  return (
```
Comment (git-marcopitra) [inline views/dapp/v2/components/layout/connect-wallet/wallet/wallet-item.tsx:15]: remove the return

File: views/dapp/v2/components/layout/connect-wallet/wallet/wallet-list-section.tsx [hunk 6]
```diff
@@ -0,0 +1,159 @@
...
+import { WalletListSectionProps } from '../connect-wallet.types';
+import WalletItem from './wallet-item';
+
+const WalletListSection: FC<WalletListSectionProps> = ({
+  setOpenWallet,
+  openWalletModal,
+}) => {
+  const t = useTranslations();
+  const { wallets } = useWalletKit();
+  const menuVariants = {
```
Comment (git-marcopitra) [inline views/dapp/v2/components/layout/connect-wallet/wallet/wallet-list-section.tsx:24]: keep it outsite

File: views/dapp/v2/components/layout/wallet/wallet.types.ts [hunk 7]
```diff
@@ -13,7 +13,8 @@ export interface WalletItemProps {
...
 }
 
-export interface WalletConnectProps {
-  isModalOpen: boolean;
-  setIsModalOpen: Dispatch<SetStateAction<boolean>>;
+export interface WalletProps {
+  setOpenConnectWallet: Dispatch<SetStateAction<boolean>>;
 }
+
+export type WalletConnectProps = WalletProps;
```
Comment (git-marcopitra) [inline views/dapp/v2/components/layout/wallet/wallet.types.ts:20]: Remove this type

**Review by git-marcopitra — CHANGES_REQUESTED** [review]

**Review by git-marcopitra — CHANGES_REQUESTED** [review]

**Pull request merged**